packages/vite/src/utils/add-vite-temp-files-to-gitignore.ts

```diff
@@ -1,7 +1,7 @@
 import { stripIndents, Tree } from '@nx/devkit';
 
 export function addViteTempFilesToGitIgnore(tree: Tree) {
-  let newGitIgnoreContents = `**/vite.config.{js,ts,mjs,mts,cjs,cts}.timestamp*`;
+  let newGitIgnoreContents = `**/vite.config.*.timestamp*`;
```

Comment:
We can skip the leading `**` and the result will be the same. As per [Git docs](https://git-scm.com/docs/gitignore#_pattern_format):

> A leading "\*\*" followed by a slash means match in all directories. For example, "**/foo" matches file or directory "foo" anywhere, the same as pattern "foo".

```suggestion
  let newGitIgnoreContents = `vite.config.*.timestamp*`;
```